### DISCUSSION TITLE
Query About Winter Of Code

### DISCUSSION THREAD

**Discussion by soumikbaithalu** [Tenet-Coding/ArtofCoding#69] [Announcements]
Hey @Harshil-Jani  should we apply to winter of Code  ???  which project we should apply to the program?

**Comment by soumikbaithalu** [comment]
@Harshil-Jani here is the [google form](https://docs.google.com/forms/d/e/1FAIpQLSdT3e31WRoxLyuYR8Ny7m7UQ0apua6luT1fzH4Zk-6ml-_y_Q/viewform) we should definitely apply to this program  ??

**Comment by Harshil-Jani** [comment]
@soumikbaithalu In the programmes like this there are generally 2 ways to register an organisation and hosting a project. Either we can allow the participants who gets selected in Tenet to work on some existing projects. But we do not have any existing projects running already. So we need to fetch some ideas while discussing and host  a project wherein people can contribute with us. It should be product based project aiming at one big  picture (A website, An Android app or Compiler building or something focusing on large scale project). Then we can fill the form. 
So we need to discuss some ideas and then act accordingly to start allowing people to contribute after winter of code session starts.

**Reply by soumikbaithalu** [reply]
okay, @Harshil-Jani  shall we discuss it ??

**Reply by Harshil-Jani** [reply]
Sure ! @soumikbaithalu
